en: update docs for br and tls (#112)
pingcap/docs-tidb-operator: en/restore-from-aws-s3-using-br.md

* `.spec.to.port`: the port of the TiDB cluster to be restored.
* `.spec.to.user`: the accessing user of the TiDB cluster to be restored.
* `.spec.to.tidbSecretName`: the secret of the user password of the `.spec.to.tidbSecretName` TiDB cluster.

> * `.spec.to.tlsClient.tlsSecret`: the secret of the storage certificate used in the restoration.

Reviewer: ```suggestion
* `.spec.to.tlsClient.tlsSecret`: the secret of the certificate used during the restoration.
```
